### ISSUE TITLE
To be production ready

### ISSUE THREAD

**Issue by stephanie-olp** [litentry/litentry-app#834]
Hey team, now that we have the beta version release, let's think about what has to be done before going on production.
Let's dress up a list of the must-have and nice to have :)

**Must have**

- [ ] 

**Nice to have**

- [ ] Find a proper solution for the feedback section in the app

**Comment by Ekaanth** [comment]
Here is a list of things that would be nice to have a user from the existing features:
- Dashboard
 	- Information about democracy and consequences of submitting a proposal, and voting
	- Descriptive information on graphs that are shown. 
	- Information about who qualifies for starting a bounty.
- Accounts
	- Provide clear information of available, locked, and contributed
	- Show the inflow and outflow along with the current market price (as in the account made 23 transactions with 234DOT in and 200DOT out with a current Balance of 34DOT) 
	- able to change to a different wallet and different network when in accounts
	- able to check their proposals, tips history, and proposed bounties
	- able to check previous transactions and redirect to polka scanner if needed.
- Registrars 
	- Doesn't have enough information to understand its purpose
- Auction 
	- More information about this would be helpful to understand
- Crowdloan
	- doesn't have information about the graph at the top
- Feedback

    - the ability to attach Images in feedback, or provide template-based feedback where the user is able to pick from like Issue, bug, or generic feedback(easy to process when the app gets scaled)

- Making a multi-lingual app would be awesome.
- Add additional resources to polka and literacy, more like the resources section.
- Most apps have terms and conditions and privacy policies. Should we mention that in the app?
- Lastly, we don't have any security on the app, Personally I feel this is a need for the app
These are some things I think would be nice to have in the app. Please feel free to correct me :))
